charts/datadog/templates/_container-agent.yaml

```diff
@@ -3,7 +3,12 @@
   image: "{{ include "image-path" (dict "root" .Values "image" .Values.agents.image) }}"
   imagePullPolicy: {{ .Values.agents.image.pullPolicy }}
   command: ["agent", "run"]
+{{- if .Values.datadog.sbom.containerImage.uncompressedLayersSupport -}}
+{{- $capabilities := dict "capabilities" (dict "add" (list "SYS_ADMIN")) }}
+{{ include "generate-security-context" (dict "securityContext" (merge $capabilities .Values.agents.containers.agent.securityContext) "targetSystem" "foobar" "seccomp" "" "kubeversion" .Capabilities.KubeVersion.Version) | indent 2 }}
+{{- else }}
```

Comment:
we already have a function `generate-security-context` that tune the securityContext. Can we add this logic to the function too?

Reply:
Done